docs/devel.md

```diff
@@ -50,12 +50,15 @@ Similarly, if you are only working on kola, ore or plume, you can build, test
 and use them directly with:
 
 ```
-$ cd mantle
-$ ./build kola ore plume
-$ ./test
+$ make kola ore plume
+$ make mantle-check
 $ ./bin/kola ...
 ```
 
+In the local developer case, you usually don't care about building
+kolet for other arches. You can skip building them using e.g.
+`make kola KOLET_ARCHES=x86_64`.
```

Comment:
hmm. Is this update re: `kolet` needed? when I type `make kola` it doesn't build `kolet`. I guess if you typed `make mantle` it would.

Reply:
Indeed, it makes more sense when doing a generic `make` instead. I moved it to a separate commit.